src/components/Books/Books.jsx

```diff
@@ -23,17 +23,36 @@ const BOOKS_QUERY = gql`
 `;
 
 export default function Books() {
+  const [showChapters, setShowChapters] = useState('');
+  function handleChapter(id) {
+    setShowChapters(id);
+  }
   const { loading, error, data } = useQuery(BOOKS_QUERY);
 
   if (loading) return <p>Loading...</p>;
   if (error) return <p>Error :(</p>;
 
-  return data.viewer.books.hits.map(({ id, displayTitle }) => (
-    <div key={id}>
-      <h3>{displayTitle}</h3>
-      <span className='chapitre'>Chapitres du livre : </span>
-      <br />
-      <Chapters bookId={id} />
-    </div>
-  ));
+  return data.viewer.books.hits
+    .filter(({ valid }) => valid === true)
```

Comment:
This is an important condition, but the test calls for a UI where the user can select a level and/or subject to only display matching books.